src/components/timeline/LayerActivity.svelte

```diff
@@ -673,7 +674,12 @@
       let y = parentY + rowHeight;
 
       for (const span of spans) {
-        const startTime = getUnixEpochTimeFromInterval(planStartTimeYmd, span.start_offset);
+        // Use simulation start YMD time if available, otherwise use the plan start YMD
+        let startYmd = planStartTimeYmd;
+        if (simulationDataset && simulationDataset.simulation_start_time) {
+          startYmd = simulationDataset.simulation_start_time;
```

Comment:
Could we simplify this line? 🥷

```ts
const startYmd = simulationDataset?.simulation_start_time ?? planStartTimeYmd;
```

Reply:
Oh yes good call. I'm still trying work nullish coalescing into my standard JS brain.